### PR TITLE
#85 fix path for Windows

### PR DESCRIPTION
**Pull request by chayn1k**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Follow the existing code style.

Also no need to open both an issue and PR. PRs are issues.

File: index.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ function md5(str) {
 }
 
 function relPath(base, filePath) {
+	if ( path.sep === '\\') {
+		base = base.replace(/\\/gi, '/');
+		filePath = filePath.replace(/\\/gi, '/');
```
Comment (bobthecow) [inline index.js]: No need for the `/i`.

Comment (bobthecow) [inline index.js]: If you're replacing all the slashes here, you should remove them from below (there are at least a couple other places that do this same replacement).

**Comment by sindresorhus** [comment]
ping @chayn1k

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**